script/elements.js

```diff
@@ -43,7 +43,7 @@ export function getFocusableElements (element = document) {
  */
 export function getKeyboardFocusableElements (element = document) {
   return [...element.querySelectorAll(
-    'a, button, input, textarea, select, details, [tabindex]:not([tabindex="-1"])'
+    'a, button, input, textarea, select, details, [tabindex]'
   )]
-    .filter(el => !el.hasAttribute('disabled'))
+    .filter(el => !el.hasAttribute('disabled') && (!el.hasAttribute('tabindex') || el.getAttribute('tabindex') >= 0))
```

Comment:
Can you help create a second `filter` statement? This one looks way too complicated :)

Reply:
Better?